### PR TITLE
community/networkmanager: update to 1.18.1

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: community/networkmanager/APKBUILD [hunk 1]
```diff
@@ -1,15 +1,14 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=networkmanager
-pkgver=1.16.0
-pkgrel=0
-pkgdesc="network management tool"
-url="http://projects.gnome.org/NetworkManager/"
+pkgver=1.18.1
+pkgrel=1
```
Comment (maxice8) [inline community/networkmanager/APKBUILD]: pkgrel should be 0

File: community/networkmanager/APKBUILD [hunk 2]
```diff
@@ -31,9 +30,10 @@ makedepends="$depends_dev
 	util-linux-dev
 	jansson-dev
 	vala
-	autoconf
-	automake
-	libtool
+	meson
+	py-gobject3
+	libpsl-dev
+	python
```
Comment (andypost) [inline community/networkmanager/APKBUILD]: Better to specify exact py3 version here

Comment (Cogitri) [reply community/networkmanager/APKBUILD]: Uh? It uses py2.

Comment (Ikke) [reply community/networkmanager/APKBUILD]: Then specify `python2`

Comment (Cogitri) [reply community/networkmanager/APKBUILD]: Ah, thanks, didn't see that!

**Pull request closed without merging**